common/src/main/java/discord4j/common/GitProperties.java

```diff
@@ -15,26 +15,38 @@
  * along with Discord4J. If not, see <http://www.gnu.org/licenses/>.
  */
 
-package discord4j.core.util;
+package discord4j.common;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.Properties;
 
-public final class VersionUtil {
+/**
+ * Provide information about the Git repository version captured at build time.
+ */
+public class GitProperties {
```

Comment:
I'm pretty sure we use singular naming throughout the project. If not we should change that. https://stackoverflow.com/a/15756009/5302085